src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/commonplatform/CommonPlatformCourtEventListenerIntTest.kt

```diff
@@ -102,7 +102,6 @@ class CommonPlatformCourtEventListenerIntTest : MessagingMultiNodeTestBase() {
       assertThat(updatedPersonEntity.getPrimaryName().sexCode).isEqualTo(changedSexCode.value)
       assertThat(updatedPersonEntity.getPnc()).isEqualTo(pnc)
       assertThat(updatedPersonEntity.getCro()).isEqualTo(cro)
-      assertThat(updatedPersonEntity.addresses.size).isEqualTo(1)
```

Comment:
this was working because the address from CP looked like this:
{
address1:  'null'
Postcode:  'null'
}

And so we were writing the string 'null' into our database. This isn't a problem in production - the data setup was incorrect